core/admin.py

```diff
@@ -30,7 +32,11 @@ class OfferingAdmin(ModelAdmin):
             'url',
             'description',
         )
-    }), ('Blackboard student list', {
+    }),
+     ('Repository settings', {
+        'fields': ('repo_url', 'repo_user', 'repo_token', 'repo_status', 'update_repository')
```

Comment:
Maybe the `repo_url` should include a help text saying that it should be the https url without username and password.